k8s/plugin/src/constant.rs

```diff
@@ -40,7 +40,7 @@ pub(crate) const UPGRADE_OPERATOR_SERVICE_PORT: i32 = 8080;
 /// Service internal port constant for upgrade operator.
 pub(crate) const UPGRADE_OPERATOR_INTERNAL_PORT: i32 = 8080;
 /// Upgrade image tag
-pub(crate) const UPGRADE_IMAGE: &str = "openebs/mayastor-operator-upgrade:develop";
+pub(crate) const UPGRADE_IMAGE: &str = "niladrih/mayastor-operator-upgrade:final3";
```

Comment:
please revert this: probably better to add a cli option or something like that to avoid merging the wrong thing

Reply:
Oops

Reply:
Fixed it, I'm not adding a cli flag for this at the moment.